app/assets/stylesheets/views/_brexit_checker_question_page.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-form-footer {
```

Comment:
In itself this feels like a bit of tech debt. This class looks like it comes from the design system (with the `.govuk-` prefix), but I can't find any record of it there. If it's not, we should rename it to make it clearer that it's only used in the checker.

Perhaps a separate card, but I think we need to look at:

1) If we actually need this any more
2) If we can replace this with something more specific

Reply:
You happy for this to live orphaned up here for the moment and then we can drop in another card for planning to fix this for good?

Reply:
Yep! Perhaps create a card and add it as a comment here?

Reply:
https://trello.com/c/i3nYBM2c/403-rename-brexit-checker-css-in-finder-frontend